src/corefx/System.Globalization.Native/CMakeLists.txt

```diff
@@ -61,9 +70,17 @@ add_library(System.Globalization.Native
 # Disable the "lib" prefix.
 set_target_properties(System.Globalization.Native PROPERTIES PREFIX "")
 
-target_link_libraries(System.Globalization.Native
-  ${ICUUC}
-  ${ICUI18N}
-)
+if(NOT CLR_CMAKE_PLATFORM_DARWIN)
+    target_link_libraries(System.Globalization.Native
+        ${ICUUC}
+        ${ICUI18N}
+    )
+else()
+    target_link_libraries(System.Globalization.Native
+        ${ICUCORE}
+    )
+
+    add_definitions(-DU_DISABLE_RENAMING=1)
```

Comment:
@nguerrera suggested that I use #if's instead of #ifdef's in in the Globalization code.  Using a cmake.configure and configure.h.in files make this really easy.  See https://github.com/dotnet/corefx/commit/61bcd097aade191538cabe4cbc13b6c8bc0c3ea3#diff-7bd8c3c1f0189b08e03c9050bca3dc61L18 for how I did it.